GithubApp/Controllers/FilterViewController.swift

```diff
@@ -9,27 +9,90 @@ import UIKit
 
 class FilterViewController: UIViewController {
 
+    //MARK: Instance Properties
+    var delegate: FilterDelegate?
+    var filterData :[Filter] = []
+    let checkedImage = UIImage(named: "check")! as UIImage
+    let uncheckedImage = UIImage(named: "uncheck")! as UIImage
+
+    // MARK: IB outlet
     @IBOutlet var filterView: FilterView!
+    
     override func viewDidLoad() {
-        super.viewDidLoad()
+        //super.viewDidLoad()
```

Comment:
Any specific reason to comment the super.viewDidLoad()?
Always call it

Reply:
Done